### PR TITLE
Create Tyj.md

### PR DESCRIPTION
**Pull request by mr975**

### CODE REVIEW

**Comment by Margen67** [comment]
@mr975 What is this?

File: Tyj.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at tyj732726@gmail.com. All
```
Comment (JoelLinn) [inline Tyj.md]: What is this email?

File: Tyj.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
```
Comment (JoelLinn) [inline Tyj.md]: Please show a little empathy and explain why you want this document added in the first place?

**Comment by MoochMcGee** [comment]
TBH, this looks like a spam account lmao

**Comment by Margen67** [comment]
Yup. There's even a randomly named repo: https://github.com/mr975/bigplaya
and another spam issue: https://github.com/jpd002/Play-/issues/863

**Pull request closed without merging**